README.md

```diff
@@ -300,6 +286,16 @@ In config page template
 {{/tableSections}}
 ```
 
+This assumes all steps containing fields have a `section` - `locals.section`, this is used to group fields in the confirm table and the email.
```

Comment:
all of our services do, they need it for displaying a title.  I guess we could improve it by catering for step config without sections too, but think that should probably be a further iteration